### PR TITLE
02. JSX, Rendering and Components

### PR DESCRIPTION
**Pull request by gladisihor**

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,11 +1,82 @@
...
+        <li>
+          <div/>
+          <p>Question.1 - Correct</p>
+        </li>
+        <li>
+          <div>hello</div>
+          <p>Question.2 - Correct</p>
+        </li>
+        <li>
+          <div>hello</div><span>World</span>
```
Comment (OkYesAnap) [inline src/App.js]: Wrong answer

File: src/App.js [hunk 2]
```diff
@@ -1,11 +1,82 @@
...
+          <p>Question.3 - Correct</p>
```
Comment (OkYesAnap) [inline src/App.js]: Wrong answer

File: src/App.js [hunk 3]
```diff
@@ -1,11 +1,82 @@
...
+        </li>
+        <li>
+          <p>Question.4 - Incorrect</p>
```
Comment (OkYesAnap) [inline src/App.js]: Wrong answer

File: src/App.js [hunk 4]
```diff
@@ -1,11 +1,82 @@
...
+          <ol>
+            <li>
+              <div>hello<span></span> World</div>
+            </li>
+            <li>
+              <div>hello
+                <span/>
+                World
+                <div/>
+              </div>
```
Comment (OkYesAnap) [inline src/App.js]: Its correct JSX... but what's the difference between this and example?

File: src/App.js [hunk 5]
```diff
@@ -1,11 +1,82 @@
...
+            &nbsp;&nbsp;&nbsp;&nbsp;"Left"{'\n'}
+            &nbsp;&nbsp;),{'\n'}
+            &nbsp;&nbsp;React.createElement({'\n'}
+            &nbsp;&nbsp;&nbsp;&nbsp;"div",{'\n'}
+            &nbsp;&nbsp;&nbsp;&nbsp;&#123;{'\n'}
+            &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;className: "SplitPane-right"{'\n'}
+            &nbsp;&nbsp;&nbsp;&nbsp;&#125;,{'\n'}
+            &nbsp;&nbsp;&nbsp;&nbsp;"Right"{'\n'}
+            &nbsp;&nbsp;){'\n'}
+          );</code>
```
Comment (OkYesAnap) [inline src/App.js]: please clear special symbols (&nbsp, '\n', &#125, &#123,  etc.). 
Use 
\`input your code here\` 
if needed.

File: src/App.js (hunk 4 above)
Comment (gladisihor) [reply src/App.js]: It is correct and means this:
React.createElement(
  "div",
  null,
  "hello ",
  React.createElement("span", null),
  " World ",
  React.createElement("div", null)
);

File: src/App.js (hunk 5 above)
Comment (gladisihor) [reply src/App.js]: updated

File: src/App.js (hunk 3 above)
Comment (gladisihor) [reply src/App.js]: fixed

File: src/App.js (hunk 2 above)
Comment (gladisihor) [reply src/App.js]: fixed

**Pull request merged**